tests/blackbox/data/evaluation/namespace-scoped/kyma_eventing_wrong_event_type/scenario.yml

```diff
@@ -0,0 +1,67 @@
+id: kyma-eventing-wrong-event-type
+description: The subscription is set-up with the wrong event type "kyma.custom.commerce.order.created.v1" instead of "sap.kyma.custom.commerce.order.created.v1".
+resource:
+  kind: Subscription
+  api_version: eventing.kyma-project.io/v1alpha2
+  name: test-v1alpha2-exact1
+  namespace: kyma-eventing-wrong-event-type
+expectations:
+  - name: subscription_config_error
+    statement: points out that the Subscription is configured with the wrong event type
+    categories:
+      - kyma
+      - problem_finding
+    complexity: 1
+  - name: subscription_correct_event_type
+    statement: provides the correct event type to use, which would be "sap.kyma.custom.commerce.order.created.v1"
+    categories:
+      - kyma
+      - solution_finding
+    complexity: 2
+  - name: subscription_explanation
+    statement: explains the use of Subscriptions and how to configure them
+    categories:
+      - kyma
+      - definition/explanation
+    complexity: 1
+  - name: step_by_step_guide
+    statement: provides a step-by-step guide on how to fix the issue
+    categories:
+      - kubernetes
+      - kyma
```

Comment:
I just noticed that this was done in the past already (e.g. [here](https://github.com/kyma-project/kyma-companion/blob/main/tests/blackbox/data/evaluation/namespace-scoped/kyma_app_serverless_syntax_err/scenario.yml)), and I will create a fix for it.

However; how about we use either `kubernetes`or `kyma`; if it is something related to 'pure' K8s, we categorize it as `kubernetes`, but if we provide fixes for Kinds that only exist in Kyma like `Subscription` or `Function`, we set it to `kyma`? 

What is your opinion?

Reply:
I agree, otherwise everything thats `kyma` would also have the tag `kubernetes`.

Reply:
As I'm already working on it, I can go ahead and also improve the other cases where both tags were applied

Reply:
Or I will create a new PR once everything is merged to fix the `kyma` and `kubernetes` tags and also align the capitalization of things like `pod` and `Pod`. What do you think?